### PR TITLE
Add button to open visit-note and fix styling

### PR DESCRIPTION
**Pull request by donaldkibet**
1. Add button to open visit-note component when no encounters have been entered.
2. Fixed styling on empty component for visit-note

### CODE REVIEW

**Review by brandones — APPROVED** [review]
LGTM, thanks!

**Comment by brandones** [comment]
Please add a screenshot, since this is new UI

File: src/widgets/notes/notes-detailed-summary.component.tsx [hunk 1]
```diff
@@ -208,17 +208,21 @@ function NotesDetailedSummary(props: NotesDetailedSummaryProps) {
...
-        styles={{
-          width: "100%",
-          backgroundColor: "var(--omrs-color-bg-low-contrast)",
-          boxShadow: "none",
-          border: "none"
-        }}
-      >
-        <div className={`${styles.emptyNotes} omrs-bold`}>
-          <p>No Notes are documented</p>
+      <SummaryCard name="Notes" styles={{ width: "100%" }}>
```
Comment (denniskigen) [inline src/widgets/notes/notes-detailed-summary.component.tsx]: Might wanna use the [EmptyState](https://github.com/openmrs/openmrs-esm-patient-chart-widgets/blob/master/src/ui-components/empty-state/empty-state.component.tsx) component here.

**Comment by brandones** [comment]
20 days old, let's get this fixed up and merged @donaldkibet

**Comment by donaldkibet** [comment]
> LGTM, thanks!

visitNote

File: src/widgets/notes/notes-detailed-summary.css [hunk 2]
```diff
@@ -79,7 +79,7 @@ button:focus {
   cursor: pointer;
 }
 
-.emptyNotes p {
+.emptyNotes {
   text-align: left;
   padding-left: 1rem;
   font-weight: 500;
```
Comment (denniskigen) [inline src/widgets/notes/notes-detailed-summary.css]: This style declaration can be safely removed as well.

**Pull request merged**